main.py

```diff
@@ -1,6 +1,9 @@
 import argparse
 import pandas as pd
 import tensorflow as tf
+
+#definition of method
+from src.Utils.tools import get_odds
```

Comment:
This import can be combined with line 
 `from src.Utils.tools import get_json_data, to_data_frame, get_todays_games_json, create_todays_games`